addon/initializers/setup-ember-can.js

```diff
@@ -0,0 +1,11 @@
+import Resolver from 'ember-resolver';
+
+Resolver.reopen({
+  // eslint-disable-next-line ember/avoid-leaking-state-in-ember-objects
```

Comment:
What makes this line leaking I wonder.

Reply:
Using {} or [] ... but in this case we dont have too much choice :/